config/config.exs

```diff
@@ -50,6 +50,12 @@ config :logger, :console,
 # Use Jason for JSON parsing in Phoenix
 config :phoenix, :json_library, Jason
 
+config :ex_aws,
+  access_key_id: System.fetch_env!("AWS_ACCESS_KEY_ID"),
+  secret_access_key: System.fetch_env!("AWS_SECRET_ACCESS_KEY"),
+  region: System.fetch_env!("AWS_REGION"),
+  request_config_override: %{}
```

Comment:
Where is this config setting documented? 
What does having an empty `Map` mean?
I can go digging to infer/understand it but I shouldn't have to ...

Reply:
no longer used. ✂️